omero/sysadmins/import-scenarios.rst

```diff
@@ -258,6 +258,52 @@ Disadvantages
   the same as the file system which the ManagedRepository lives on. This
   prevents the use of network file systems and similar remote shares.
 
+
+.. _parallel_import:
+
+Parallel import
+---------------
+
+Parallel import is an experimental variant of manual CLI import for
+making large-scale imports considerably faster. This feature entails
+*significant risk* and may be changed or removed at any time. If high
+thread counts are specified then the import client or OMERO server may
+stop working. It is therefore imperative that new uses of parallel
```

Comment:
"stop working"

This doesn't have more risk that other parallelism and we've even taken steps to improve it, so this is likely too strong.